march_hardware_interface/src/march_hardware_interface_node.cpp

```diff
@@ -21,10 +21,13 @@ int main(int argc, char** argv)
   AllowedRobot selected_robot = AllowedRobot(argv[1]);
   ROS_INFO_STREAM("Selected robot: " << selected_robot);
 
+  bool reset_imc = ros::param::param<bool>("~reset_imc", 1);
+  ROS_INFO_STREAM("Use the reset of the IMC: " << reset_imc);
```

Comment:
The resetting is also logged when it happens, so this seems redundant and only necessary for debugging.